test/e2e_node/services/kubelet.go

```diff
@@ -190,6 +197,58 @@ func createPodManifestDirectory() (string, error) {
 	return path, nil
 }
 
+// createKubeconfig creates a kubeconfig file at the fully qualified `path`.
+func createKubeconfig(path string) error {
+	kubeconfig := `apiVersion: v1
+kind: Config
+users:
+- name: kubelet
+clusters:
+- cluster:
+    server: ` + getAPIServerClientURL() + `
+    insecure-skip-tls-verify: true
```

Comment:
@Random-Liu is the intention to eventually do this, or is this not a big deal because this suite will only ever connect over localhost? I'd be scared if we were worried about something sniffing lo...